### PR TITLE
Enable user_status by default

### PR DESCRIPTION
**Pull request by georgehrke**
Follow up for #21186

### CODE REVIEW

**Review by juliusknorr — APPROVED** [review]

**Review by jancborchardt — APPROVED** [review]
Yuss please

File: apps/user_status/appinfo/info.xml [hunk 1]
```diff
@@ -25,6 +25,7 @@
     <dependencies>
         <nextcloud min-version="20" max-version="20"/>
     </dependencies>
+    <default_enable />
```
Comment (nickvergessen) [inline apps/user_status/appinfo/info.xml:28]: It's already in line 12

File: core/shipped.json [hunk 2]
```diff
@@ -40,6 +40,7 @@
     "twofactor_backupcodes",
     "updatenotification",
     "user_ldap",
+    "user_status",
```
Comment (nickvergessen) [inline core/shipped.json:43]: Dashboard is also missing in this list

**Review by nickvergessen — APPROVED** [review]

File: core/shipped.json (hunk 2 above)
Comment (jancborchardt) [reply core/shipped.json:44]: Ref @juliushaertl for adding that :)

Comment (nickvergessen) [reply core/shipped.json:44]: Already done

**Pull request merged**